scripts/CEE/cordon-drain-reboot-worker-node/README.md

```diff
@@ -0,0 +1,48 @@
+# Cordon/Uncordon / Drain / Reboot worker nodes
+
+## Description
+
+This script is responsible for performing Cordoning/Uncordoning, Draining and Rebooting operations on Worker nodes, giving the options to perform such operations as required.
+
+## Available Options
+
+The managed script provides the following actions:
+
+- cordon: To cordon worker nodes
+- uncordon: To uncordon worker nodes
+- drain: This will cordon and then drain worker nodes
+- reboot: This will cordon, drain and then reboot worker nodes
+
+## Usage
+
+**ATTENTION** ⚠️ This script must only be used by the MCS (Managed Cloud Services) CRU (Crew Response Unit) team.
+- If you need this script to be used, contact a MCS CRU member.
+- The usage of this script is audited.
+
+### Cordon/Uncordon
+
+```bash
+ocm backplane managedjob create CEE/cordon-drain-reboot-worker-node -p WORKER="<node_name>" -p ACTION="[cordon|uncordon]"
```

Comment:
I am thinking maybe we can give it a more common name, like `worker-node-operations`

Reply:
Yes, @bmeng , the idea is to deprecate those duplicate.
And about the name, `worker-node-operations` is not bad, but it will not be clear which operations we are referring to, just on the description. But you guys decide which is better.

Reply:
I would give a +1 to the suggestion above. The operations are already mentioned in the Readme file.

Reply:
I was just thinking, if we have more operations to add here, and the script name would be no sense again, and we may need to decide either to create a new script for replacement or create a separated script for that part.

Reply:
Shall I change it back, @bmeng ?
The name it was does not matter as it will only describe what it is and be copied and pasted.